app/models/user.rb

```diff
@@ -1,8 +1,17 @@
 class User < ApplicationRecord
-  has_secure_password
+  has_secure_password validations: false, if: "uid"
+  validates :email, confirmation: true, if: "uid.nil?", on: :create
```

Comment:
We might want to make sure there is some validation when omniauth users log in. (e.g. validate for uid, first_name, email and oauth_token)